### ISSUE TITLE
Translation error in every minikube command since 1.15.0

### ISSUE THREAD

**Issue by galishmann** [kubernetes/minikube#9704]
When I am trying to execute any command in minikube I have error message:
```sh
~ $ minikube version
I1114 13:59:47.843637   11252 translate.go:89] Failed to load translation file for en: Asset translations/en.json not found
minikube version: v1.15.0
commit: 3e098ff146b8502f597849dfda420a2fa4fa43f0
```

**Comment by afbjorklund** [comment]
Works here, which platform are you on ?

**Comment by afbjorklund** [comment]
Hmm, can reproduce with non-american locale:

```
$ LC_ALL=en_US.UTF-8 ./out/minikube version
minikube version: v1.15.0
commit: 3e098ff146b8502f597849dfda420a2fa4fa43f0
$ LC_ALL=en_UK.UTF-8 ./out/minikube version
I1114 14:50:45.479064  444106 translate.go:89] Failed to load translation file for en: Asset translations/en.json not found
minikube version: v1.15.0
commit: 3e098ff146b8502f597849dfda420a2fa4fa43f0
```
Or for other languages:
```
$ LC_ALL=sv_SE.UTF-8 ./out/minikube version
I1114 14:52:27.157482  444194 translate.go:89] Failed to load translation file for sv: Asset translations/sv.json not found
minikube version: v1.15.0
commit: 3e098ff146b8502f597849dfda420a2fa4fa43f
```

Worked OK in v1.14.2.

**Comment by afbjorklund** [comment]
No changes to translate itself, though. So it looks like a `klog` change.

**Comment by fnichol** [comment]
I can confirm this behavior with the 1.15.0 release (running macOS 10.15.7).

Explicitly setting an `en_US` locale:

```console
$ env LC_ALL=en_US.UTF-8 minikube version
minikube version: v1.15.0
commit: 3e098ff146b8502f597849dfda420a2fa4fa43f0
```

Setting a non-US locale--mine in this case:

```console
$ env LC_ALL=en_CA.UTF-8 minikube version
I1114 13:07:25.983227   66526 translate.go:89] Failed to load translation file for en: Asset translations/en.json not found
minikube version: v1.15.0
commit: 3e098ff146b8502f597849dfda420a2fa4fa43f0
```

No locale environment variable set, presumably falling back to a variant of `en_CA` in my case:

```console
$ env -u LC_ALL minikube version
I1114 13:06:48.939471   66485 translate.go:89] Failed to load translation file for en: Asset translations/en.json not found
minikube version: v1.15.0
commit: 3e098ff146b8502f597849dfda420a2fa4fa43f0
```

---

As always, I appreciate everyone's hard work on minikube! ❤️

**Comment by afbjorklund** [comment]
Before, we had these two lines in the minikube log:

```
translate.go:107] Setting Language to en-GB ...
translate.go:89] Failed to load translation file for en: Asset translations/en.json not found
```

Now the first line is gone, the other is on the console:

* Points to 20a1274c7c0d493494a9e0c437c7958610d8f7b2
* Reverting this fixes it

**Comment by afbjorklund** [comment]
The error message is extra useless, since it doesn't actually look at that path (adding the file doesn't help)

So it only looks at the files that have been compiled into minikube the binary, and are available as an Asset

**Comment by sharifelgamal** [comment]
This is fixed in v1.15.1
